swagger.yaml

```diff
@@ -876,6 +876,7 @@ paths:
       tags:
         - manifests
       security:
+        - agentAuth: []
```

Comment:
Why is this needed?

Reply:
for tests, as i understand rest will be disabled on crd and i need this api to be able to validate subsystem

Reply:
agent auth will not disabled on CRD...
I am not a fan of changing security for tests. Any other way to validate?

Reply:
@tsorya the REST API is getting tested in subsystem.

Reply:
@rollandf thats why i added agent auth. 
I don't see any other option to validate it from subsystem to say the truth. If you have idea will be glad to hear

Reply:
If the agent is not calling this API don't add agentAuth

Reply:
yeah, knew it :) .
not sure what to do with subsystem though, will we allow user api calls in subsystem? @filanov ?